protocol/avdl/kbfs.avdl

```diff
@@ -6,8 +6,25 @@
 
 protocol kbfs {
 
+  import idl "common.avdl";
   import idl "kbfs_common.avdl";
 
+  record TLFMetadataNotification {
+      UID creator;
+      UID lastWriter;
+      string folderName;
+      string folderID;
+      long folderRevision;
+
+      long usageBytes;
+
+      // Members of the TLF
+      array<UID> resolvedWriters;
+      array<UID> resolvedReaders;
+      array<string> unresolvedWriters;
+      array<string> unresolvedReaders;
```

Comment:
I wonder if it's wasteful to be resending these on every update in a steady state.  Maybe only need to send on a change?